blocks/library/inserter/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { keycodes } from '@wordpress/utils';
+
+/**
+ * Internal dependencies
+ */
+import './block.scss';
+import { registerBlockType, createBlock } from '../../api';
+import InserterMenu from '../../../editor/inserter/menu';
+
+const { BACKSPACE, ESCAPE } = keycodes;
+
+registerBlockType( 'core/inserter', {
```

Comment:
I've not tested this, but what's the expected behavior with this being implemented as a block? A user has to insert the inserter block, then they have the option to `/` to trigger the menu? I'd not really expected this to be a block, but rather something built-in to either Editable or the default paragraph block.